src/ArrowEffects.cpp

```diff
@@ -94,21 +94,27 @@ static float GetNoteFieldHeight()
 
 float ArrowEffects::GetTime()
 {
-	float mult = 1.f + curr_options->m_fModTimerMult;
-	float offset = curr_options->m_fModTimerOffset;
+	double mult = 1.0 + static_cast<double>(curr_options->m_fModTimerMult);
+	double offset = static_cast<double>(curr_options->m_fModTimerOffset);
 	ModTimerType modtimer = curr_options->m_ModTimerType;
+	double time;
 	switch(modtimer)
 	{
-	    case ModTimerType_Default:
-	    case ModTimerType_Game:
-		return (RageTimer::GetTimeSinceStart()+offset)*mult;
-	    case ModTimerType_Beat:
-		return (GAMESTATE->m_Position.m_fSongBeatVisible+offset)*mult;
-	    case ModTimerType_Song:
-		return (GAMESTATE->m_Position.m_fMusicSeconds+offset)*mult;
-	    default:
-		return RageTimer::GetTimeSinceStart()+offset;
-	}
+		case ModTimerType_Default:
+		case ModTimerType_Game:
+			time = (RageTimer::GetTimeSinceStart() + offset) * mult;
```

Comment:
Let's call it something other than `time`. It's not particularly a keyword, but it is a c-function. Also just initialize to some default value as well.